src/chapters/html/semantic-html.rst

```diff
@@ -314,25 +314,3 @@ Check Your Understanding
 
 .. Answers = block, block, inline, block, block, block, block, inline
 
-.. raw:: html
-
-   <script type="text/JavaScript">
-      function evaluateMC(id, correct) {
-         if (correct) {
-            document.getElementById(id).innerHTML = 'Yep! Tag names do not affect loading speed.';
-            document.getElementById(id).style.color = 'blue';
-         } else {
-            document.getElementById(id).innerHTML = 'Nope!';
-            document.getElementById(id).style.color = 'red';
-         }
-      }
```

Comment:
This is another example of a question-specific feedback message for a multiple choice question.  Let's keep this function here, but the `revealAnswer` function is OK to remove.

Reply:
added back in

Reply:
added back in